### PR TITLE
Issue #282 - Hide menubar in Windows 10

### PR DESCRIPTION
**Pull request by montrueque**

### CODE REVIEW

**Comment by perry-mitchell** [comment]
Closes #282.

@sallar Is this how we want it? I guess it should be toggle-able. There are many important functions within the menu that are not accessible otherwise, and we wouldn't want to hinder usability.

File: src/main/windows.js [hunk 1]
```diff
@@ -24,6 +24,8 @@ export function setupWindows() {
       vibrancy: 'ultra-dark'
     });
 
+    win.setMenuBarVisibility(false);
```
Comment (perry-mitchell) [inline src/main/windows.js]: @sallar Should this not make it hidden always?

Comment (sallar) [reply src/main/windows.js]: @perry-mitchell if this is set to `false`, the user can still bring the menu up by pressing `alt`. From the docs:

> Sets whether the menu bar should be visible. If the menu bar is auto-hide, users can still bring up the menu bar by pressing the single Alt key.

But I personally thing we should make this toggle-able by having a menu item in View menu for example lables "Auto Hide Menubar" which should only be available in windows.

Comment (perry-mitchell) [reply src/main/windows.js]: Agreed. Even having it default to shown (first install) and being able to hide it _later_ with alt would be preferable.

**Comment by sallar** [comment]
Thanks @montrueque for the contribution. I took your PR and built on top of it #337. Cheers.

**Pull request merged**